cypress/integration/login.js

```diff
@@ -88,5 +81,37 @@ describe('Sourcebooks login', function() {
         }
 
         rolesArray.forEach(testFunction);        
+    })    
+})
+
+describe('Sourcebook admin tasks', function() {

```

Comment:
You should move this whole task describe block another file like `task.js`, because it looks weird having task tests in `login.js `file next to login test cases